File: android/app/src/main/java/com/phicdy/totoanticipation/model/GameListStorage.java
package com.phicdy.totoanticipation.model;

import android.support.annotation.NonNull;

import java.util.List;

public interface GameListStorage {
    List<Game> list(@NonNull String totoNum);
    void store(@NonNull String totoNum, @NonNull List<Game> list);
}

File: android/legacy/src/main/java/com/phicdy/totoanticipation/legacy/model/storage/GameListStorage.kt
package com.phicdy.totoanticipation.legacy.model.storage

import com.phicdy.totoanticipation.domain.Game
import com.phicdy.totoanticipation.domain.Toto
import java.util.Date

interface GameListStorage {
    fun totoNum(): String
    fun totoDeadline(): Date
    fun list(totoNum: String): List<Game>
    fun store(toto: Toto, list: List<Game>)
}
